README.md

```diff
@@ -1,33 +1,31 @@
-# Operate First template for repositories
+# Project Meteor
 
-Derive new repositories from this template
+Similar to how a meteoroid entering the atmosphere, becomes visible as a meteor and finally hitting the Earth's surface as a meteorite, we aim to create better visibility and impact of our team's work through project *Meteor*.
 
-List of featurese:
+It is an AICoE combined team effort to provide a product for Data Scientists and other users alike where they can interact and leverage our services, tools and technology for developing various applications. The product can also help in introducing, sharing and promoting our team's capabilities with others.
 
-## License
+## Product
```

Comment:
Do we want to add anything about how this project is supported by the Operate First environment? Or better to keep it agnostic?

Reply:
I think we can mention that, ill add it in

Reply:
is "product" the right word? Doesn't Red Hat have a pretty serious distinction between "projects" and "products" maybe we call it a tool?